contrib/terraform/vsphere/modules/kubernetes-cluster/variables.tf

```diff
@@ -0,0 +1,36 @@
+## Global ##
+variable "prefix" {}
+
+variable "machines" {
+  description = "Cluster machines"
+  type = map(object({
+    node_type = string
+    ip      = string
+  }))
+}
+
+variable "gateway" {}
```

Comment:
Can you check with `terraform fmt`? I think newer Terraform asks to set a type for each variable.